WINDOWS.md

```diff
@@ -1,5 +1,33 @@
 # Windows
 
+## Steps to build Windows image.
+
+### Requirements:
+* Windows host with build version equal o grater than the target os version of the image. (eg: To build a an image for 20H2 version the machine needs to be 20H2) 
+* Docker for Windows installed
+
+### Build base image (Infrastructure Agent)
+* Clone the Infra Agent repo from github:
+  ```
+  git clone git@github.com:newrelic/infrastructure-agent.git
+  ```
+* Get the latest Agent binaries from [newrelic](http://download.newrelic.com/infrastructure_agent/binaries/windows/amd64/) and and place them in `infrastructure-agent/target/bin/windows_amd64/`
```

Comment:
Is there a reason for not we automating this step, either in the Dockerfile or in the ps1? Likely out of scope already, but doesn't feel good to read instructions asking you to manually download binaries.

Reply:
I would automate the process taking the agent image from the docker hub instead of automating this manual process. But right now the agent windows image is not being pushed to the hub